docs/UserGuide.md

```diff
@@ -420,7 +428,8 @@ Views all session grades of a student within the focused class.
 " %}
 
 Format: `view INDEX`
-* Views the grade of the student at index `INDEX` for the currently focused class.
+* Views the grades of the student at index `INDEX` for the currently focused class.
```

Comment:
"Views" is probably not the right word here. Maybe "Shows", "Displays"?

Reply:
That sounds like a good choice!